package.json

```diff
@@ -19,17 +19,20 @@
         "styles"
     ],
     "scripts": {
+        "prepare": "husky install",
         "test": "exit 0",
         "clean": "gulp clean",
         "build": "gulp",
         "start": "start-storybook -p 7007",
         "build-storybook": "build-storybook -c .storybook -o storybook-static",
         "lint:js": "eslint --ext .js,.jsx,.ts,.tsx .",
         "lint:styles": "stylelint '{styles,src}/**/*.scss'",
-        "lint:prettier": "prettier --check --loglevel=error '**/*.{js,jsx,ts,tsx,scss}'",
+        "lint:prettier": "prettier --check --loglevel=error '**/*.md'",
         "lint": "run-p lint:*",
         "typecheck": "tsc --noEmit",
-        "prepublishOnly": "npm run build"
+        "prepublishOnly": "npm run build",
+        "pre-commit": "npx nano-staged",
+        "commit-msg": "npx commitlint --edit $1"
```

Comment:
Let's use commands directly from husky scripts. There is no practical benefit of listing them in npm scripts

Reply:
fixed